src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -313,7 +313,7 @@ class HoldConfirmation extends React.Component {
             for <Link id="item-link" to={`${appConfig.baseUrl}/bib/${bibId}`}>{title}</Link>
           </p>
           <p id="delivery-location">
-            The item will be delivered to: {this.renderLocationInfo(deliveryLocation)}
+            The item will be delivered to{pickupLocation === 'edd' ? ' the email address you provided.' : [': ', this.renderLocationInfo(deliveryLocation)]}
```

Comment:
In the second part of that ternary, should there be a `.join` or are arrays of strings automatically joined by '' in react?

Reply:
React can take an array of elements, which can be strings, HTML, or React elements. `renderLocationInfo` returns HTML. 

I'm reworking this to move more logic into the function so the code reads a little more explicitly (in my personal opinion)